lib/flash_flow/issue_tracker/pivotal.rb

```diff
@@ -151,6 +151,10 @@ def removed_branches
         @branches.select { |_, v| v.removed? }
       end
 
+      def merged_working_branch
+        merged_branches.select { |_, branch| branch.ref == @git.working_branch }
```

Comment:
Since this can only be one branch, I'd do this instead:
```
merged_branches.detect { |_, branch| branch.ref == @git.working_branch }
```

That will either return the branch or nil, instead of returning an array. It's a little confusing to use `select` when you know at most there's going to be one match.